gitops/octopus/OctopusDeploy.md

```diff
@@ -0,0 +1,170 @@
+# Octopus Deploy
+
+Octopus Deploy, an automated deployment and release management tool, has been tested against the Bedrock GitOps workflow. There are many differences between using Octopus Deploy as opposed to Azure DevOps. This README will assist in getting you started on installing, configuring, and deploying a Release on an Octopus Server hosted in Azure.
```

Comment:
Exactly what kind of [pipeline](https://github.com/Microsoft/bedrock/blob/f9ef3c559ad8061ecdb2d9e6ac5a369128ede3cc/gitops/PipelineThinking.md) will this guide be helping people create?

Reply:
I now see on the last sentence of the next `Getting Started` section we explain that this will be a manifest generation pipeline. We may want to move that up